_content/_content.json

```diff
@@ -1,3 +1,3 @@
 { 
-    "layout": "../_layouts/main.njk"
+    "layout": "_layouts/main.njk"
```

Comment:
Since `_content/` is your input directory, I think we discussed that this won't work for setting a global layout default until a new version of Eleventy is pushed to npm with that recent PR. Also, this should probably just have been "main.njk" since we already define the `_layouts/` directory path above in our .eleventy.js file (line 19 above).